package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "babel-eslint": ">=8.2.0",
     "eslint-config-prettier": ">=2.9.0",
-    "eslint-plugin-flowtype": ">=2.46.0",
+    "eslint-plugin-flowtype": "^2.49.3",
```

Comment:
Maybe keep the range signifier as `>=`?

Reply:
Sure! I just ran `npm install --save eslint-plugin-flowtype@latest`